### PR TITLE
[Sec-9] Adding entities for skills + lombok + sql create scripts

### PR DESCRIPTION
**Pull request by gizet**

### CODE REVIEW

File: src/main/resources/database/changes/0.0/create_tables_secur_skills.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS SKILL_TYPE
+(
+    id         bigint NOT NULL AUTO_INCREMENT,
+    SKILL_TYPE varchar(255) DEFAULT NULL,
```
Comment (CosminaIacob) [inline src/main/resources/database/changes/0.0/create_tables_secur_skills.sql]: remove default null, please

File: src/main/resources/database/changelog-skills.xml [hunk 2]
```diff
@@ -4,9 +4,9 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <!--    Example of how to include new script. TODO delete this after first script-->
-    <!--    <changeSet id="1" author="test">-->
-    <!--        <sqlFile path="/database/changelog/changes/test.sql"/>-->
-    <!--    </changeSet>-->
+    <!--SPRINT 0-->
+    <changeSet id="2" author="CatalinLu">
```
Comment (CosminaIacob) [inline src/main/resources/database/changelog-skills.xml]: start with 1 for id, please

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,7 +1,8 @@
-server.port=8081
-spring.datasource.url=jdbc:mysql://localhost:3306/secur_skills
+spring.datasource.url=jdbc:mysql://localhost:3306/secur_skills?createDatabaseIfNotExist=true
 spring.datasource.username=root
 spring.datasource.password=password
+server.port=8081
```
Comment (CosminaIacob) [inline src/main/resources/application.properties]: add server to the first line please, and when merge with dev pay attention to add it in the properties file for local profile

File: src/main/java/ro/secur/skillsdata/entity/SkillEntity.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+@Getter
+@Setter
+@Table(name = "SKILL")
+@Entity
+public class SkillEntity implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
```
Comment (CosminaIacob) [inline src/main/java/ro/secur/skillsdata/entity/SkillEntity.java]: remove @Column, please

File: src/main/resources/application.properties (hunk 3 above)
Comment (gizet) [reply src/main/resources/application.properties]: done

File: src/main/resources/database/changes/0.0/create_tables_secur_skills.sql [hunk 5]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS SKILL_TYPE
+(
+    id         bigint NOT NULL AUTO_INCREMENT,
```
Comment (ftansanu) [inline src/main/resources/database/changes/0.0/create_tables_secur_skills.sql]: id -> change it to uppercase :)))

Comment (gizet) [reply src/main/resources/database/changes/0.0/create_tables_secur_skills.sql]: done

File: src/main/resources/database/changes/0.0/create_tables_secur_skills.sql (hunk 1 above)
Comment (gizet) [reply src/main/resources/database/changes/0.0/create_tables_secur_skills.sql]: done

File: src/main/java/ro/secur/skillsdata/entity/SkillEntity.java (hunk 4 above)
Comment (gizet) [reply src/main/java/ro/secur/skillsdata/entity/SkillEntity.java]: done

**Pull request merged**